dashboard/lib/tasks/seed.rake

```diff
@@ -164,13 +164,9 @@ namespace :seed do
     'step',
     'oceans',
     'sports',
-  ].map {|script| "config/scripts/#{script}.script"}.freeze
+  ].map {|script| "config/scripts_json/#{script}.script_json"}.freeze
   SEEDED = 'config/scripts/.seeded'.freeze
 
-  file SEEDED => [SCRIPTS_GLOB, :environment].flatten do
```

Comment:
it took me a while to figure out what this rule was and confirm that it was unused. it creates a task that can be executed via `rake config/scripts/.seeded` (without `seed:` namespace). 
* official docs:  https://ruby.github.io/rake/doc/rakefile_rdoc.html#label-File+Tasks 
* how to call it: https://avdi.codes/rake-part-1-basics/
* why no namespace: https://ruby.github.io/rake/doc/rakefile_rdoc.html#label-FileTasks